lib/README.md

```diff
@@ -1,13 +0,0 @@
-_This lib folder contains external libs that are not provided by their author with Maven._
-
-**(The following libraries located in this folder have been approved by their author!)**
-
-PAFSpigot (Simonsator#5834 | https://www.spigotmc.org/resources/authors/simonsator.67188/ )
-
-
-**(The libraries on the subfolder `/spigot/`, have been released in compliance with the license therein)**
-
-According to the **GNU General Public License v3.0 and GNU Lesser General Public License v3.0** it is allowed to distribute the software.
-
-
-If you are the owner of one of these libraries feel free to contact us if you want removal of your lib ( https://discord.plugily.xyz or management@plugily.xyz ) . 
```

Comment:
nice attempt to remove the jar but its needed for import net.minecraft.server.v1_8_R3.X